src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+
+    /**
+     * Returns a printable string representation of the object.
+     */
+    String getPrintableString();
+
+    /**
+     * Returns if printable is private
+     */
+    default boolean isPrivate() { return false; }

```

Comment:
Coding standard violation:exclamation: A single line method should not be written in a single line. See point 5 [[Java Coding Standard: Comments](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#layout)]